rcl/test/rcl/test_wait.cpp

```diff
@@ -179,11 +181,11 @@ TEST(CLASSNAME(WaitSetTestFixture, RMW_IMPLEMENTATION), multi_wait_set_threaded)
             // printf("%s\n", ss.str().c_str());
           }
           ret = rcl_wait(&test_set.wait_set, wait_period);
+          ASSERT_NE(ret, RCL_RET_ERROR);
```

Comment:
My comment about this still stands. As-is it would miss a problem if, for example, `RCL_RET_INVALID_ARGUMENT` was returned, which is an error but not `RCL_RET_ERROR`. The safe thing to check is "if not RCL_RET_TIMEOUT or RCL_RET_OK then error occurred". Thanks for catching this gap in the logic though.